src/upload.ts

```diff
@@ -190,14 +187,22 @@ async function uploadVideo(videoJSON: Video, messageTransport: MessageTransport)
         progress = { progress: 0, stage: ProgressEnum.Processing }
         videoJSON.onProgress(progress)
     }
-
     if (videoJSON.onProgress) {
         clearInterval(progressChecker)
         progressChecker = undefined
         progress = { progress: 100, stage: ProgressEnum.Done }
         videoJSON.onProgress(progress)
     }
 
+    try {
+        if (videoJSON.forKid) {
+            await page.click("tp-yt-paper-radio-button[name='VIDEO_MADE_FOR_KIDS_MFK']")
+        } else {
+            await page.click("tp-yt-paper-radio-button[name='VIDEO_MADE_FOR_KIDS_NOT_MFK']")
+        }
+    } catch {
+    }
+
```

Comment:
- Try catch covers too much code : It's only required around each of the await page.click calls
- We have a logger "messageTransport.log('message')" that you could use to tell any user using that logger that this thing gone wrong
- If I'm not mistaken, this made for kids option is what the option this package used to require its user to set up a default value for their accounts. Which means that people using this package with the default value as "made for kids", if they don't pass this new parameters, they'll end up with "not made for kids" on their videos. I think we'd want not to set anything if this value is keep as undefined, and switch that value is we pass a boolean.

Reply:
I use try catch empty because i want to skip this action if kid selection don't exist. 
I don't known some how this Kid selection is Required in my upload channel.
Because i think for kid options must be intentional enable. Just in case people upload inappropriate to yt.

Reply:
anyway , thank for reviewing my PR.